cmd/generate/generate.go

```diff
@@ -103,6 +110,7 @@ func main() {
 	pflag.StringVar(&imagesFromRepositoriesURLFmt, "images-from-url-format", "https://raw.githubusercontent.com/openshift-knative/%s/%s/openshift/images.yaml", "Additional images to be pulled from other midstream repositories matching the tag in project.yaml")
 	pflag.StringArrayVar(&additionalPackages, "additional-packages", nil, "Additional packages to be installed in the image")
 	pflag.StringVar(&templateName, "template-name", defaultDockerfileTemplateName, fmt.Sprintf("Dockerfile template name to use. Supported values are [%s, %s]", defaultDockerfileTemplateName, funcUtilDockerfileTemplateName))
+	pflag.BoolVar(&rpmsLockFileEnabled, "generate-rpms-lock-file", false, "Enable the creation of the rpms.lock.yaml file")
```

Comment:
Is there a reason, why you add this as a parameter? Isn't this only relevant for func? And not sure if we should use "dedicated" lock files for the different components - or would you for simplicity use one big, which is used by multiple components?

Reply:
for must gather

Reply:
dedicated lockfile is not worth it, let's optimize when we actually need to, currently, we only have a few RPMs across the board

Reply:
:+1: